src/Login/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import "./Login.css";
+import {toast, ToastContainer} from "react-toastify";
+import LoginForm from "./LoginForm";
+import "../common/Notify.css"
+
+function Login() {
+
+    const url = window.location.href;
```

Comment:
We are using [react-router-dom](https://reactrouter.com/web/api/Hooks) so we should avoid accessing the routing props from the windows. 

Check the link of react-router-dom and try to use it. It has some cool hooks to work with react.
In this case, you should be doing something like this:
```
import {useLocation} from "react-router-dom";

// inside you login component
const location = useLocation();
```